services/aldryn/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "name": "aldryn",
+    "label": "Divio Aldryn",
+    "description": "Lift django CMS into the cloud"
+  },
+  "fields": [
+    {
+      "name": "project_domain",
+      "label": "Aldryn Project Domain",
+      "description": "e.g mysite.us.aldryn.io",
+      "example": "mysite.us.aldryn.io"
+    }
+  ],
+  "records": [
+    {
+      "type": "ALIAS",
+      "content": "{{project_domain}}.aldryn.io",
```

Comment:
Because of the way you're prompting for the value (asking for the whole project domain and giving an example of `mysite.us.aldryn.io`), this should just be `"content": "{{project_domain}}",` The same goes for the `www` record. If you'd like to prompt for just the project name and zone so you can enforce that the record is always added for an `aldryn.io` URL, you should change the example above to be `mysite.us`.